<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMakeMainEditor</class>
 <widget class="XUPPageEditor" name="QMakeMainEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../../monkey/src/resources/resources.qrc">
    <normaloff>:/xupeditor/icons/xupeditor/project.png</normaloff>:/xupeditor/icons/xupeditor/project.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbProjectType">
     <property name="title">
      <string>Project Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbStaticLibrary">
        <property name="text">
         <string>Static Library</string>
        </property>
        <attribute name="buttonGroup">
         <string>bgType</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rbQtPlugin">
        <property name="text">
         <string>Qt Plugin</string>
        </property>
        <attribute name="buttonGroup">
         <string>bgType</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbSharedLibrary">
        <property name="text">
         <string>Shared Library</string>
        </property>
        <attribute name="buttonGroup">
         <string>bgType</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rbQtDesignerPlugin">
        <property name="text">
         <string>Qt Designer Plugin</string>
        </property>
        <attribute name="buttonGroup">
         <string>bgType</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbApplication">
        <property name="text">
         <string>Application</string>
        </property>
        <attribute name="buttonGroup">
         <string>bgType</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbSolution">
        <property name="text">
         <string>Solution</string>
        </property>
        <attribute name="buttonGroup">
         <string>bgType</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbProjectInformations">
     <property name="title">
      <string>Informations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lProjectName">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>leProjectName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="leProjectName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lProjectTarget">
        <property name="text">
         <string>Target directory</string>
        </property>
        <property name="buddy">
         <cstring>leProjectTarget</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leProjectTarget"/>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="tbProjectTarget">
        <property name="icon">
         <iconset resource="../../../../../monkey/src/resources/resources.qrc">
          <normaloff>:/listeditor/icons/listeditor/folder.png</normaloff>:/listeditor/icons/listeditor/folder.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbQtVersion">
     <property name="title">
      <string>Qt Version</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" rowspan="2">
       <widget class="QListWidget" name="lwQtVersion">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../../../monkey/src/resources/resources.qrc">:/xupeditor/icons/xupeditor/about.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QLabel" name="lQtVersion">
           <property name="text">
            <string>Select the Qt version used by the project.
The Qt versions can be managed in the QMake plugin configuration.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>8</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XUPPageEditor</class>
   <extends>QFrame</extends>
   <header>XUPPageEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../monkey/src/resources/resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="bgType"/>
 </buttongroups>
</ui>
